zrml/swaps/src/fixed.rs

```diff
@@ -53,7 +55,7 @@ pub fn bsub_sign(a: u128, b: u128) -> Result<(u128, bool), DispatchError> {
 }
 
 pub fn bpowi(a: u128, n: u128) -> Result<u128, DispatchError> {
-    let mut z = if n % 2 != 0 { a } else { BASE };
```

Comment:
Oh, severe that we used `%` before 😰. But good reminder that `number % 0` produces a panic.

Reply:
I wonder, if we have other places using the `%` operator.. In court we have one for sure, but no zero division is possible there..

Reply:
Could you update the court pallet regarding this change too in this PR?

Reply:
#1254; reverted changes from this PR